test/lit.cfg

```diff
@@ -396,7 +396,7 @@ if test_sdk_overlay_dir is not None:
         "-L %s -Xlinker -rpath -Xlinker %s" %
         (sdk_overlay_link_path_dir, sdk_overlay_link_path_dir))
     lit_config.note('Using SDK overlay dir: ' + test_sdk_overlay_dir)
-    resource_dir_opt += (" %s" % sdk_overlay_dir_opt)
+    config.resource_dir_opt += (" %s" % sdk_overlay_dir_opt)
```

Comment:
These tools (swift-ide-test, swift-refactor, sourcekitd-test) have a different CLI than the driver, and only handle certain options.  I suggest we split this `-L %s -Xlinker -rpath -Xlinker %s ...` into a separate variable in lit so that we can pass only `-resource-dir %s` to these tools.

Reply:
Ok. I can fix this in a follow on PR. It isn't directly related to this.